core/src/main/java/de/danoeh/antennapod/core/storage/DBReader.java

```diff
@@ -811,14 +812,18 @@ public static StatisticsResult getStatistics(boolean includeMarkedAsPlayed,
      * items.
      */
     @NonNull
-    public static NavDrawerData getNavDrawerData() {
+    public static NavDrawerData getNavDrawerData(boolean ignoreFilter) {
```

Comment:
Instead of a parameter "ignoreFilter", I think it would be nicer to use the filter itself as a parameter. Then that's one more step into the direction of decoupling database access and user preferences.

Reply:
You mean a variable of type `SubscriptionsFilter`? Or some lambda function that does the filtering?

Reply:
Yes, SubscriptionsFilter